web/static/css/_extends/_forms.sass

```diff
@@ -87,7 +87,8 @@
       box-shadow : 0 8px 6px -6px darken($gray, 20%)
 
   button.thesis-tray-cancel,
-  button.thesis-tray-save
+  button.thesis-tray-save,
```

Comment:
below are a ton of css styles that rework the editor buttons to make them more future-proof. also new styles for the import/export tray.